app/controllers/concerns/formatting_concern.rb

```diff
@@ -14,14 +14,27 @@ def records_to_xml_string(records)
         writer.close()
       end
       VoyagerHelpers::Liberator.valid_xml(xml_str)
+    elsif records.kind_of? (String)
```

Comment:
It seems like formatting might be better off living in the adapters, rather than as a controller concern. Especially since we're using a conditional on what type of object we have as a proxy for which adapter we were using.

Something consider for a future PR, maybe.